public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -20,11 +21,15 @@ export const DashboardEmpty = ({ dashboard, canCreate }: Props) => {
       <div className={cx(styles.centeredContent, styles.wrapper)}>
         <div className={cx(styles.containerBox, styles.centeredContent, styles.visualizationContainer)}>
           <h1 className={cx(styles.headerSection, styles.headerBig)}>
-            Start your new dashboard by adding a visualization
+            <Trans i18nKey="dashboard.empty.add-visualization.header">
```

Comment:
Same in these ones, reduce from 4 segments to 3 🙏

Reply:
`dashboard-empty.add-visualization.header` or `dashboard.empty.add-visualization-header`